lib/zhr_devs/bakery_integration/command_runner.ex

```diff
@@ -1,8 +1,15 @@
 defmodule ZhrDevs.BakeryIntegration.CommandRunner do
   @moduledoc """
   Generic process that will run the given command
+
+  To run successfully the command must implement the ZhrDevs.BakeryIntegration.Commands.Command behaviour
+
+  on_success is zero arity function that will be called when the command finishes successfully.
+  on_failure is one arity function that will be called when the command fails.
+
+  If at any point in time process terminates abnormally it would be restarted, because it's transient.
```

Comment:
So currently we'll be just looping bad submissions, right?

Reply:
I don't understand what this comment means. I was testing on real submissions that are provided in `zhr_bakery/submissions` folder, which is not bad submissions, as they build and run successfully.
One of them is yield 0 result all the time, but... It runs.